ods_ci/tests/Tests/100__deploy/130__operators/150__dsp_operator/150__dsp_operator.robot

```diff
@@ -0,0 +1,72 @@
+*** Settings ***
+Documentation       Test Cases with regards to the DSP Operator
+
+Library             Collections
+Library             SeleniumLibrary
+Library             OpenShiftLibrary
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource            ../../../../Resources/OCP.resource
+Resource            ../../../../Resources/RHOSi.resource
+Resource            ../../tasks/Resources/RHODS_OLM/uninstall/uninstall.robot
+
+Suite Setup         Suite Setup
+Suite Teardown      Suite Teardown
+
+
+*** Test Cases ***
+Detect Pre-existing Install Of Argo Workflows And Block RHOAI Install
+    [Documentation]    Detect Pre-existing Install Of Argo Workflows And Block RHOAI Install
+    [Tags]                  Operator                ODS-2651                Tier3
+    Delete Argo Workflow Crd
+    ${return_code}          ${output}               Run And Return Rc And Output
+    ...                     oc apply -f ./ods_ci/tests/Resources/Files/argo/crd.workflows.yaml
+    Should Be Equal As Integers
+    ...                     ${return_code}
+    ...                     0
+    ...                     msg=${output}
+    Can Uninstall RHODS Operator
+    Should Be Equal As Integers ${return_code}      0
+    Open OperatorHub
+    Install ODH Operator
```

Comment:
Uninstall RHODS but installing ODH?

Reply:
This logic is being replaced with the olm install script execution on upcoming commit

Reply:
ok but still, if we need to merge this PR before that change, I'ld like to understand: are we re-installing ODH instead of RHOAI? if yes, is that ok?